assets/src/js/admin/admin-setup.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Accessible Block Links
+ *
+ * Problem: Hyperlink a component while maintaining screen-reader accessibility and the ability to select text.
+ * Solution: Use progressive enhancement to conditionally trigger the target anchor element.
+ *
+ * @link https://css-tricks.com/block-links-the-search-for-a-perfect-solution/
```

Comment:
Great idea linking to this article! Really helpful in understanding the thinking behind this functionality/implementation